utilities/transactions/transaction_test.cc

```diff
@@ -9453,6 +9455,37 @@ TEST_P(CommitBypassMemtableTest, ThresholdTxnDBOption) {
   }
 }
 
+TEST_P(CommitBypassMemtableTest, AtomicFlushTest) {
+  const uint32_t threshold = 10;
+  SetUpTransactionDB(/*threshold=*/threshold, /*atomic_flush=*/true);
+  SyncPoint::GetInstance()->EnableProcessing();
+
+  std::vector<std::string> cfs = {"cf0", "cf1", "cf2"};
+  CreateColumnFamilies(cfs, options);
+
+  // Seed data in CF1 and 2 as atomic flush picks CFs with non-empty memtable
+  ASSERT_OK(db_->Put({}, handles_[1], "key1", "val1"));
+  ASSERT_OK(db_->Put({}, handles_[2], "key2", "val2"));
+
+  // Write to cf 1, should see 2 and 3 flushed too
```

Comment:
nit: 1, 2, 3 here is refering to CF0, CF1, CF2 as above documentation, right? Maybe call they the same?